crates/omnix-health/src/check/shell.rs

```diff
@@ -87,7 +88,7 @@ struct CurrentUserShellEnv {
     /// Current shell
     shell: Shell,
     /// *Absolute* paths to the dotfiles
-    dotfiles: HashMap<&'static str, PathBuf>,
+    dotfiles: BTreeMap<&'static str, PathBuf>,
```

Comment:
Out of scope for this PR, but that's okay.

Reply:
Ah, wrong find and replace. I will revert it